source/exe/BUILD

```diff
@@ -10,9 +10,16 @@ load(
 
 envoy_package()
 
+alias(
+    name = "envoy",
+    actual = ":envoy-static",
+    visibility = ["//visibility:public"],
+)
+
 envoy_cc_binary(
     name = "envoy-static",
     repository = "@envoy",
+    visibility = ["//visibility:public"],
```

Comment:
why does this become public visibility?

Reply:
I was throwing spaghetti with commits aba6fc6, afdf822, and 57d1e14. The latter is the one that unblocked so the former two can probably be reverted